delfin/drivers/utils/ssh_client.py

```diff
@@ -147,7 +148,10 @@ def __init__(self, **kwargs):
         self.conn_timeout = self.SOCKET_TIMEOUT
         if self.ssh_conn_timeout is None:
             self.ssh_conn_timeout = SSHPool.SOCKET_TIMEOUT
-        super(SSHPool, self).__init__(min_size=0, max_size=3)
+        ssh_max_size = kwargs.get('ssh_max_size')
+        if ssh_max_size is None:
+            ssh_max_size = self.MAX_POOL_SIZE
```

Comment:
These three lines 151,152,153 can be written as `ssh_max_size = kwargs.get('ssh_max_size', self.MAX_POOL_SIZE)` .

Reply:
done